### ISSUE TITLE
About the LINEMOD dataset test set and training set

### ISSUE THREAD

**Issue by Sunweijun333** [liyi14/mx-DeepIM#47]
I would like to ask that the division of test and train in your LINEMOD dataset and the LINEMOD I downloaded is slightly different. I want to know how to divide training and data in your dataset.

**Comment by wangg12** [comment]
It is the same as the split in the BB8 paper (which is available in the google drive provided in our REAME.)
